vehicle_gateway_integration_test/test/test_px4.py

```diff
@@ -81,7 +81,7 @@ def generate_test_description():
         output='screen')
 
     gz_args = '-r -v 4 empty_px4_world.sdf'
-    #gz_args = '--headless-rendering -s -r -v 4 empty_px4_world.sdf'
+    # gz_args = '--headless-rendering -s -r -v 4 empty_px4_world.sdf'
```

Comment:
I would go ahead and remove line 83 and re-add the code at line 84 (with `--headless-rendering`). If I had to guess this was an accidental commit